yarn.lock

```diff
@@ -2,6 +2,14 @@
 # yarn lockfile v1
```

Comment:
Should I be committing the yarn.lock file?

Reply:
I'd say probably not in this case as none of the dependencies have changed